frontends/common/setup.h

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Logically belongs with the preprocessor, which is in the frontend, but currently in options.h/options.cpp.  We could perhaps just put these two declarations in options.h and get rid of this file, if that is not too obscure.